MediaManager.Forms/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,5 @@
+﻿using Xamarin.Forms;
+
+[assembly: XmlnsPrefix("http://baseflow.com/mediamanager", "media")]
```

Comment:
MediaManager uses SDK-style project and will generate the assembly info. This makes the build fail i think. Could you change the name into `AssemblyInfoAttributes.cs`? Just like: https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross.Forms/AssemblyInfoAttributes.cs

Reply:
SDK Style shouldn't have anything to do with that. By convention an SDK style projects generate the file with all of the base Version/Meta-Data attributes as {ProjectName}.AssemblyInfo.cs in the Intermediate Output Folder (i.e. `obj/{Configuration}`)… That said looking at that build output from AppVeyor it seems that they're doing something really weird that is causing a conflict.

Reply:
I had the same problem with MvvmCross and am pretty sure that if you rename the file it will work.

Reply:
No problem.. like I mentioned it looks to me that AppVeyor was not happy with it. I renamed this per your suggestion